code/src/components/MainPage.tsx

```diff
@@ -1,30 +1,530 @@
+                in sem a nisl wsde faucibus lobortis.
+              </p>
+            </div>
+            <div className="w-9/12 absolute -translate-y-6 translate-x-90 px-14">
+              <div className="w-full flex justify-between items-center h-25 -translate-y-10">
+                <div className="w-2 h-2 bg-extra-light-grey rounded-full translate-x-2" />
+                <div className="flex-grow w-full h-0.5 bg-extra-light-grey mx-2" />
+                <div className="w-2 h-2 bg-extra-light-grey rounded-full -translate-x-2" />
+              </div>
+            </div>
+          </div>
+          {/* Video element with rounded corners */}
+          <div className="w-8/12 px-14">
+            {/* Video element */}
+            <video
+              width="100%"
+              height="100%"
+              controls
+              aria-label="Description of the video's content"
+              title="Short description of the video"
+              className="rounded-4xl"
+            >
+              <source src="/sample.mp4" type="video/mp4" />
+              {/* Adjusting 'kind' attribute to "captions" */}
+              <track
+                kind="captions"
+                src="/subtitle_en.vtt"
+                srcLang="en"
+                label="English"
+                default
+              />
+              Your browser does not support the video tag.
+            </video>
+          </div>
+        </section>
 
-      <div className="text-white p-40 text-5xl font-bold">
-        <p>Keep track of</p>
-        <p>your rental</p>
+        {/* Footer */}
+        <footer className="relative flex flex-col justify-center items-center gap-20 rounded-t-7xl bg-black text-white z-30 p-6 mx-14">
+          <div className="w-5/12 mb-4 justify-center items-center pt-10 text-center">
+            <p className="text-5xl">
+              Lorem ipsum dolor sit amet, adipiscing elit.
+            </p>
+          </div>
+          <button
+            type="button"
+            className="bg-custom-orange text-white hover:bg-custom-hover-orange transition duration-800 px-4 ease-in-out rounded-3xl py-2 sm:py-4 sm:px-8"
+          >
+            Join <span className="font-bold">RentalCompany</span> Now
+          </button>
+          {/* Company Logo */}
+          <Image
+            src="/CompanyLogo.svg"
+            alt="Company Logo"
+            width={366}
+            height={36}
+          />
+          <div className="w-full flex flex-row justify-evenly items-center">
+            <p className="w-1/3 text-sm text-center text-custom-grey">
+              Terms & Agreements
+            </p>
+            <p className="w-1/3 text-sm text-center text-custom-grey">
+              @2023 RentalCompany All Rights Reserved
+            </p>
+            <p className="w-1/3 text-sm text-center text-custom-grey">
+              Privacy Policy
+            </p>
+          </div>
+        </footer>
```

Comment:
The component should consist of 500 lines of code, anything over 200 is very bad. Need to split it up and try to reuse code. Other components are also too long

Reply:
You copy pasted old version of AccessPage component, you should just use a new version of AccessPage instead of your 500 lines file and update it as you need